sentry-samples/sentry-samples-jul/src/main/java/io/sentry/samples/jul/Main.java

```diff
@@ -2,18 +2,22 @@
 
 import java.util.UUID;
 import java.util.logging.Level;
+import java.util.logging.LogManager;
 import java.util.logging.Logger;
 import org.slf4j.MDC;
 
 public class Main {
 
   private static final Logger LOGGER = Logger.getLogger(Main.class.getName());
 
-  public static void main(String[] args) {
+  public static void main(String[] args) throws Exception {
+    LogManager.getLogManager()
```

Comment:
I think there should be a comment that this is made only for the purpose of running it from an IDE.

Reply:
Added a comment for an alternative way of providing the `logging.properties` file.